frontend/src/assets/main.css

```diff
@@ -1,35 +1,59 @@
 @import './base.css';
 
```

Comment:
not sure where to place this comment, the menu looks amazing!
If we will be using the lined square maybe its nice to make is a bit more in the same ratio as the design? (a bit more squarisch but not fully)

Reply:
which lined square? I did nothing with a lines square 🙃

Reply:
<img width="1806" alt="Screen Shot 2023-01-30 at 3 26 47 PM" src="https://user-images.githubusercontent.com/58222549/215504266-0b464ae5-d7f3-4776-8cec-70dfe61187a7.png">

Reply:
deze rechthoek rondom this is a login page

Reply:
I guess that is indeed part of this first setup thingy :P 
I honestly just made a box, not really worrying about the size, but I adjusted it now to be 4/5 aspect ratio (which is close to what it is in penpot)

Reply:
perfect!